### PR TITLE
remove residual use of options obsoleted by #1266

### LINKED ISSUE #1237: Don't expose password in process listing
## Description
The service will use the authentication password as a command line option (`--secret=`).
This will allow **any** user on this system to obtain the password from the process table (`ps ...`)

---

## Current behavior
The service will use the authentication password as a command line option (`--user_password=`).
This will allow **any** user on this system to obtain the password from the process table (`ps ...`)

---

## Possible Solution
I haven't touched Java for years, but if I'm recalling correctly the command line arguments  are not immutable in Java (they aren't in C/Go/Rust).

Unlike "classic" UNIX systems the process arguments on Linux are not read only. Changing the relevant argument to some obfuscated value (after processing them) will be reflected in the process output of the operating system.

This can be used to prevent access to the password/shared secrets to anyone running `ps`.

### PR DESCRIPTION
**Pull request by obfusk**
Closes #1237 (thanks to #1266).

- [x] CLA signed :)

### CODE REVIEW

File: resources/mvn-exec.sh [hunk 1]
```diff
@@ -8,4 +8,4 @@ fi
 
 . $CONFIG
 
-exec mvn compile exec:exec -Dexec.executable=java -Dexec.args="-cp %classpath ${JAVA_SYS_PROPS} org.jitsi.videobridge.Main --domain=\"${JVB_HOSTNAME}\" --host=\"${JVB_HOST}\" --port=\"${JVB_PORT}\" --secret=\"${JVB_SECRET}\" --opts=${JVB_OPTS}"
+exec mvn compile exec:exec -Dexec.executable=java -Dexec.args="-cp %classpath ${JAVA_SYS_PROPS} org.jitsi.videobridge.Main --opts=${JVB_OPTS}"
```
Comment (obfusk) [inline resources/mvn-exec.sh:11]: Should `--opts=${JVB_OPTS}` just be `${JVB_OPTS}` or am I missing something? I don't see a `--opts` defined anywhere.

Comment (bgrozev) [reply resources/mvn-exec.sh:11]: Yes, I think so, good catch

Comment (obfusk) [reply resources/mvn-exec.sh]: I added a commit to fix that.

**Comment by bgrozev** [comment]
Thank you for the contribution!

**Pull request merged**